### PR TITLE
oncall docs refactor

### PR DESCRIPTION
**Pull request by alyssawada**
# What this PR does

- Restructure the OnCall documentation to align with Grafana Labs [documentation structure guidance](https://grafana.com/docs/writers-toolkit/structure/).  

- Updates documentation front matter to include all required fields.

## Which issue(s) this PR fixes
Related to issue #3700 

## Checklist

- [ ] Unit, integration, and e2e (if applicable) tests updated
- [ ] Documentation added (or `pr:no public docs` PR label added if not required)
- [ ] `CHANGELOG.md` updated (or `pr:no changelog` PR label added if not required)

### CODE REVIEW

File: docs/sources/manage/mobile-app/on-call-status-and-shifts/index.md [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
+  - OnCall
+  - Mobile app
+  - Shift
+  - Calendar
+  - Schedules
+  - Status
+canonical: https://grafana.com/docs/oncall/latest/manage/mobile-app/on-call-status-and-shifts/
+aliases:
+  - /docs/grafana-cloud/alerting-and-irm/oncall/manage/mobile-app/on-call-status-and-shifts/
+  - /docs/grafana-cloud/alerting-and-irm/oncall/mobile-app/on-call-status-and-shifts/
```
Comment (jdbaldry) [inline docs/sources/manage/mobile-app/on-call-status-and-shifts/index.md:16]:
```suggestion
  - /docs/grafana-cloud/alerting-and-irm/oncall/manage/mobile-app/on-call-status-and-shifts/
  - /docs/grafana-cloud/alerting-and-irm/oncall/mobile-app/on-call-status-and-shifts/
  - ../../mobile-app/on-call-status-and-shifts/ # /docs/oncall/<ONCALL_VERSION>/mobile-app/on-call-status-and-shifts/
```

File: docs/sources/manage/mobile-app/push-notifications/index.md [hunk 2]
```diff
@@ -1,12 +1,19 @@
...
+  - OnCall
+  - Mobile app
+  - iOS
+  - Android
+  - Push notification
+  - Override
+canonical: https://grafana.com/docs/oncall/latest/manage/mobile-app/push-notifications/
+aliases:
+  - /docs/grafana-cloud/alerting-and-irm/oncall/manage/mobile-app/push-notifications/
+  - /docs/grafana-cloud/alerting-and-irm/oncall/mobile-app/push-notifications/
```
Comment (jdbaldry) [inline docs/sources/manage/mobile-app/push-notifications/index.md:16]:
```suggestion
  - /docs/grafana-cloud/alerting-and-irm/oncall/manage/mobile-app/push-notifications/
  - /docs/grafana-cloud/alerting-and-irm/oncall/mobile-app/push-notifications/
  - ../../mobile-app/push-notifications/ # /docs/oncall/<ONCALL_VERSION>/mobile-app/push-notifications/
```

File: docs/sources/manage/notify/_index.md [hunk 3]
```diff
@@ -1,18 +1,20 @@
...
-weight: 800
+  - ChatOps
+  - Slack
+  - MS Teams
+  - Telegram
+  - Phone
+canonical: https://grafana.com/docs/oncall/latest/manage/notify/
+aliases:
+  - /docs/grafana-cloud/alerting-and-irm/oncall/manage/notify/
+  - /docs/grafana-cloud/alerting-and-irm/oncall/notify/
```
Comment (jdbaldry) [inline docs/sources/manage/notify/_index.md:17]:
```suggestion
  - /docs/grafana-cloud/alerting-and-irm/oncall/manage/notify/
  - /docs/grafana-cloud/alerting-and-irm/oncall/notify/
  - ../notify/ # /docs/oncall/<ONCALL_VERSION>/notify/
```

File: docs/sources/manage/notify/ms-teams/index.md [hunk 4]
```diff
@@ -1,18 +1,20 @@
...
+  - MS Teams
   - Microsoft
-title: Microsoft Teams
-weight: 500
+  - Channels
+canonical: https://grafana.com/docs/oncall/latest/manage/notify/ms-teams/
+aliases:
+  - /docs/grafana-cloud/alerting-and-irm/oncall/manage/notify/ms-teams/
+  - /docs/grafana-cloud/alerting-and-irm/oncall/notify/ms-teams
+  - ../../chat-options/configure-teams/
```
Comment (jdbaldry) [inline docs/sources/manage/notify/ms-teams/index.md:17]:
```suggestion
  - /docs/grafana-cloud/alerting-and-irm/oncall/manage/notify/ms-teams/
  - /docs/grafana-cloud/alerting-and-irm/oncall/notify/ms-teams
  - ../../chat-options/configure-teams/ # /docs/oncall/<ONCALL_VERSION>/chat-options/configure-teams/
  - ../../notify/ms-teams/ # /docs/oncall/<ONCALL_VERSION>/notify/ms-teams/
```

File: docs/sources/manage/notify/phone-calls-sms/index.md [hunk 5]
```diff
@@ -1,18 +1,18 @@
...
-  - slack
-title: Phone calls and SMS
-weight: 100
+  - SMS
+  - Phone
+  - Rate Limits
+canonical: https://grafana.com/docs/oncall/latest/manage/notify/phone-calls-sms/
+aliases:
+  - /docs/grafana-cloud/alerting-and-irm/oncall/manage/notify/phone-calls-sms/
+  - /docs/grafana-cloud/alerting-and-irm/oncall/notify/phone-calls-sms
```
Comment (jdbaldry) [inline docs/sources/manage/notify/phone-calls-sms/index.md:15]:
```suggestion
  - /docs/grafana-cloud/alerting-and-irm/oncall/manage/notify/phone-calls-sms/
  - /docs/grafana-cloud/alerting-and-irm/oncall/notify/phone-calls-sms/
   - ../../notify/phone-sms/ # /docs/oncall/<ONCALL_VERSION>/notify/phone-sms/
   - ../../notify/phone-calls-sms/ # /docs/oncall/<ONCALL_VERSION>/notify/phone-calls-sms/
```

File: docs/sources/manage/notify/slack/index.md [hunk 6]
```diff
@@ -1,17 +1,19 @@
...
-title: Slack
-weight: 100
+  - ChatOps
+  - Slack
+  - Channels
+canonical: https://grafana.com/docs/oncall/latest/manage/notify/slack/
+aliases:
+  - /docs/grafana-cloud/alerting-and-irm/oncall/manage/notify/slack/
+  - /docs/grafana-cloud/alerting-and-irm/oncall/notify/slack/
+  - ../../chat-options/configure-slack/
```
Comment (jdbaldry) [inline docs/sources/manage/notify/slack/index.md:16]:
```suggestion
  - /docs/grafana-cloud/alerting-and-irm/oncall/manage/notify/slack/
  - /docs/grafana-cloud/alerting-and-irm/oncall/notify/slack/
  - ../../chat-options/configure-slack/ # /docs/oncall/<ONCALL_VERSION>/chat-options/configure-slack/
  - ../../notify/slack/ # /docs/oncall/<ONCALL_VERSION>/notify/slack/
```

File: docs/sources/manage/notify/telegram/index.md [hunk 7]
```diff
@@ -1,17 +1,19 @@
...
-title: Telegram
-weight: 300
+  - ChatOps
+  - Telegram
+  - Channels
+canonical: https://grafana.com/docs/oncall/latest/manage/notify/telegram/
+aliases:
+  - /docs/grafana-cloud/alerting-and-irm/oncall/manage/notify/telegram/
+  - /docs/grafana-cloud/alerting-and-irm/oncall/notify/telegram/
+  - ../../chat-options/configure-telegram/
```
Comment (jdbaldry) [inline docs/sources/manage/notify/telegram/index.md:16]:
```suggestion
  - /docs/grafana-cloud/alerting-and-irm/oncall/manage/notify/telegram/
  - /docs/grafana-cloud/alerting-and-irm/oncall/notify/telegram/
  - ../../chat-options/configure-telegram/ # /docs/oncall/<ONCALL_VERSION>/chat-options/configure-telegram/
  - ../../notify/telegram/ # /docs/oncall/<ONCALL_VERSION>/notify/telegram/
```

File: docs/sources/manage/on-call-schedules/_index.md [hunk 8]
```diff
@@ -1,16 +1,19 @@
...
+  - On-call
+  - Schedules
+  - Rotation
+  - Calendar
+  - Shift
+  - Scheduler
+canonical: https://grafana.com/docs/oncall/latest/manage/on-call-schedules/
+aliases:
+  - /docs/grafana-cloud/alerting-and-irm/oncall/manage/on-call-schedules/
+  - /docs/grafana-cloud/alerting-and-irm/oncall/on-call-schedules/
```
Comment (jdbaldry) [inline docs/sources/manage/on-call-schedules/_index.md:16]:
```suggestion
  - /docs/grafana-cloud/alerting-and-irm/oncall/manage/on-call-schedules/
  - /docs/grafana-cloud/alerting-and-irm/oncall/on-call-schedules/
  - ../on-call-schedules # /docs/oncall/<ONCALL_VERSION>/on-call-schedules/
```

File: docs/sources/set-up/open-source/index.md [hunk 9]
```diff
@@ -1,8 +1,17 @@
...
-title: Open Source
-weight: 400
+  - OnCall
+  - Open source
+  - OSS
+  - Install
+canonical: https://grafana.com/docs/oncall/latest/set-up/open-source/
+aliases:
+  - /docs/grafana-cloud/alerting-and-irm/oncall/set-up/open-source/
+  - /docs/oncall/latest/open-source/
```
Comment (jdbaldry) [inline docs/sources/set-up/open-source/index.md:14]:
```suggestion
  - /docs/grafana-cloud/alerting-and-irm/oncall/set-up/open-source/
  - /docs/grafana-cloud/alerting-and-irm/oncall/open-source/
  - ../open-source/ # /docs/oncall/<ONCALL_VERSION>/open-source/
```

File: docs/sources/set-up/migration-from-other-tools/index.md [hunk 10]
```diff
@@ -1,13 +1,17 @@
...
+weight: 200
+keywords:
+  - OnCall
+  - Migration
+  - Pagerduty
+  - on-call tools
+canonical: https://grafana.com/docs/oncall/latest/set-up/migration-from-other-tools/
 aliases:
+  - /docs/grafana-cloud/alerting-and-irm/oncall/set-up/migration-from-other-tools
   - /docs/oncall/latest/migration-from-other-tools/
```
Comment (jdbaldry) [inline docs/sources/set-up/migration-from-other-tools/index.md:14]:
```suggestion
  - /docs/grafana-cloud/alerting-and-irm/oncall/set-up/migration-from-other-tools
  - /docs/grafana-cloud/alerting-and-irm/oncall/migration-from-other-tools
  - ../migration-from-other-tools/ # /docs/oncall/<ONCALL_VERSION>/migration-from-other-tools/
```

File: docs/sources/set-up/get-started/index.md [hunk 11]
```diff
@@ -1,13 +1,20 @@
...
+  - OnCall
+  - Configure
+  - Escalation chains
+  - Integrations
+  - On-call schedules
+canonical: https://grafana.com/docs/oncall/latest/set-up/get-started/
+aliases:
+  - /docs/grafana-cloud/alerting-and-irm/oncall/set-up/get-started/
+  - /docs/oncall/latest/get-started/
+  - ../getting-started/
```
Comment (jdbaldry) [inline docs/sources/set-up/get-started/index.md:17]:
```suggestion
  - /docs/grafana-cloud/alerting-and-irm/oncall/set-up/get-started/
  - /docs/grafana-cloud/alerting-and-irm/oncall//get-started/
  - ../getting-started/ # /docs/oncall/<ONCALL_VERSION>/getting-started/
  - ../get-started/ # /docs/oncall/<ONCALL_VERSION>/get-started/
```

File: docs/sources/oncall-api-reference/_index.md [hunk 12]
```diff
@@ -1,7 +1,17 @@
...
+weight: 900
+keywords:
+  - OnCall
+  - API
+  - HTTP
+  - API key
+canonical: https://grafana.com/docs/oncall/latest/oncall-api-reference/
+aliases:
+  - /docs/grafana-cloud/alerting-and-irm/oncall/oncall-api-reference/
+  - /docs/grafana-cloud/alerting-and-irm/oncall/
```
Comment (jdbaldry) [inline docs/sources/oncall-api-reference/_index.md:14]:
```suggestion
  - /docs/grafana-cloud/alerting-and-irm/oncall/oncall-api-reference/
```

File: docs/sources/manage/user-and-team-management/_index.md [hunk 13]
```diff
@@ -1,11 +1,17 @@
...
+  - Notifications
   - RBAC
-  - permissions
-  - notification
-weight: 1300
+  - Permissions
+  - Roles
+canonical: https://grafana.com/docs/oncall/latest/manage/user-and-team-management/
+aliases:
+  - /docs/grafana-cloud/alerting-and-irm/oncall/manage/user-and-team-management/
```
Comment (jdbaldry) [inline docs/sources/manage/user-and-team-management/_index.md:14]:
```suggestion
  - /docs/grafana-cloud/alerting-and-irm/oncall/manage/user-and-team-management/
  - /docs/grafana-cloud/alerting-and-irm/oncall/user-and-team-management/
  - ../user-and-team-management/ # /docs/oncall/<ONCALL_VERSION>/user-and-team-management/
```

File: docs/sources/manage/on-call-schedules/api-terraform-schedule/_index.md [hunk 14]
```diff
@@ -1,14 +1,19 @@
...
+  - On-call
+  - Schedules
+  - Rotation
+  - Calendar
+  - Terraform
+  - as-code
+canonical: https://grafana.com/docs/oncall/latest/manage/on-call-schedules/api-terraform-schedules/
+aliases:
+  - /docs/grafana-cloud/alerting-and-irm/oncall/manage/on-call-schedules/api-terraform-schedules/
+  - /docs/grafana-cloud/alerting-and-irm/oncall/on-call-schedules/api-terraform-schedules/
```
Comment (jdbaldry) [inline docs/sources/manage/on-call-schedules/api-terraform-schedule/_index.md:16]:
```suggestion
  - /docs/grafana-cloud/alerting-and-irm/oncall/manage/on-call-schedules/api-terraform-schedules/
  - /docs/grafana-cloud/alerting-and-irm/oncall/on-call-schedules/api-terraform-schedules/
  - ../../on-call-schedules/api-terraform-schedule/ # /docs/oncall/<ONCALL_VERSION>/on-call-schedules/api-terraform-schedule/
```

File: docs/sources/manage/on-call-schedules/ical-schedules/index.md [hunk 15]
```diff
@@ -1,14 +1,19 @@
...
+  - On-call
+  - Schedules
+  - Rotation
+  - Calendar
+  - iCal
+  - Import
+canonical: https://grafana.com/docs/oncall/latest/manage/on-call-schedules/ical-schedules/
+aliases:
+  - /docs/grafana-cloud/alerting-and-irm/oncall/manage/on-call-schedules/ical-schedules/
+  - /docs/grafana-cloud/alerting-and-irm/oncall/on-call-schedules/ical-schedules/
```
Comment (jdbaldry) [inline docs/sources/manage/on-call-schedules/ical-schedules/index.md:16]:
```suggestion
  - /docs/grafana-cloud/alerting-and-irm/oncall/manage/on-call-schedules/ical-schedules/
  - /docs/grafana-cloud/alerting-and-irm/oncall/on-call-schedules/ical-schedules/
  - ../../on-call-schedules/ical-schedules/ # /docs/oncall/<ONCALL_VERSION>/on-call-schedules/ical-schedules/
```

File: docs/sources/manage/on-call-schedules/shift-swaps/_index.md [hunk 16]
```diff
@@ -1,14 +1,19 @@
...
+  - On-call
+  - Schedules
+  - Rotation
+  - Calendar
+  - Shift
+  - Swap
+canonical: https://grafana.com/docs/oncall/latest/manage/on-call-schedules/shift-swaps/
+aliases:
+  - /docs/grafana-cloud/alerting-and-irm/oncall/manage/on-call-schedules/shift-swaps/
+  - /docs/grafana-cloud/alerting-and-irm/oncall/on-call-schedules/shift-swaps/
```
Comment (jdbaldry) [inline docs/sources/manage/on-call-schedules/shift-swaps/_index.md:16]:
```suggestion
  - /docs/grafana-cloud/alerting-and-irm/oncall/manage/on-call-schedules/shift-swaps/
  - /docs/grafana-cloud/alerting-and-irm/oncall/on-call-schedules/shift-swaps/
  - ../../on-call-schedules/shift-swaps/ # /docs/oncall/<ONCALL_VERSION>/on-call-schedules/shift-swaps/
```

File: docs/sources/manage/on-call-schedules/web-schedule/_index.md [hunk 17]
```diff
@@ -1,14 +1,17 @@
...
+weight: 300
+keywords:
+  - On-call
+  - Schedules
+  - Rotation
+  - Calendar
+canonical: https://grafana.com/docs/oncall/latest/manage/on-call-schedules/web-schedule/
+aliases:
+  - /docs/grafana-cloud/alerting-and-irm/oncall/manage/on-call-schedules/web-schedule/
+  - /docs/grafana-cloud/alerting-and-irm/oncall/on-call-schedules/web-schedule/
```
Comment (jdbaldry) [inline docs/sources/manage/on-call-schedules/web-schedule/_index.md:14]:
```suggestion
  - /docs/grafana-cloud/alerting-and-irm/oncall/manage/on-call-schedules/web-schedule/
  - /docs/grafana-cloud/alerting-and-irm/oncall/on-call-schedules/web-schedule/
  - ../../on-call-schedules/web-schedule/ # /docs/oncall/<ONCALL_VERSION>/on-call-schedules/web-schedule/
```

**Review by jdbaldry — COMMENTED** [review]
Aliases are so fun!

**Comment by jdbaldry** [comment]
My review was so long that some of the comments have been hidden, do try and apply them all :)

**Comment by alyssawada** [comment]
Hey @jdbaldry! I'm still working through doc-validator errors but otherwise, this about does it for phase 1 of the restructure. Let me know if you see anything that seems out of place though. Thanks again!

**Comment by alyssawada** [comment]
Hey @mackgorski Here is the first stage of the OnCall restructure - All feedback is welcome! 
As a reminder, this PR establishes the structure needed. Updates and improvements to existing content is out of scope for this stage.

**Comment by jdbaldry** [comment]
> Hey @jdbaldry! I'm still working through doc-validator errors but otherwise, this about does it for phase 1 of the restructure. Let me know if you see anything that seems out of place though. Thanks again!

Cool! I just made sure that the "latest" `doc-validator` tag was up to date and updated the script to provide you more human friendly output.

Give me a shout if any of the output of `make doc-validator` is confusing!

**Comment by mackgorski** [comment]
Left two minor comments/questions, other than that looks great!!!

**Review by mackgorski — APPROVED** [review]
Looks great to me. I left to minor non-blocking comments, just to be sure to check them out. But other than that great job!

File: docs/sources/intro/_index.md [hunk 18]
```diff
@@ -0,0 +1,20 @@
+---
+title: Introduction to Grafana OnCall
+menuTitle: Introduction
+description: An introduction to Grafana OnCall including, an overview of key concepts and features, how it works, and what you can do with it.
+weight: 100
+keywords:
+  - OnCall
+  - Alert notifications
+  - Escalation chains
+  - On-call schedules
```
[the remark this answers is not in the document]
Comment (alyssawada) [reply docs/sources/intro/_index.md:10]: Fair point, Mack! "OnCall" refers to the proper product name, Grafana OnCall. Then "on-call" with the hyphen is used when referring to on-call schedules. But overall, these keywords are used for SEO purposes so it doesn't hurt to have both :)

**Comment by alyssawada** [comment]
@jdbaldry working through conflicts now :)

**Review by jdbaldry — APPROVED** [review]
I've not reviewed the prose but I think we had fixed most if not all links yesterday and the aliases were checked a while before.

**Review by brendamuir — APPROVED** [review]

**Pull request merged**